libkbfs/folder_branch_status.go

```diff
@@ -172,11 +175,23 @@ func (fbsk *folderBranchStatusKeeper) getStatus(ctx context.Context) (
 		fbs.RekeyPending = fbsk.config.RekeyQueue().IsRekeyPending(fbsk.md.ID)
 		fbs.FolderID = fbsk.md.ID.String()
 		fbs.Revision = fbsk.md.Revision
+
+		jServer, err := GetJournalServer(fbsk.config)
```

Comment:
Polling it on demand is ok for now, but maybe add a TODO here that ideally the journal would push a status update to us, so that we can close the `StatusUpdate` channel and let listeners know, so they don't have to poll.

And maybe add to the function comment that journal status changes won't cause the channel to close?